nautobot/extras/views.py

```diff
@@ -453,39 +453,26 @@ class CustomFieldDeleteView(generic.ObjectDeleteView):
     queryset = CustomField.objects.all()
 
 
-class CeleryTaskFailure(Exception):
-    """
-    Custom Exception for when a celery task failed to execute.
-    Attributes:
-        obj_name -- The obj name for which the celery task failed to delete
-        message -- explanation of the error
-    """
-
-    def __init__(self, obj_name=None, message="Celery task has failed for custom field "):
-        self.message = message
-        self.message += f"{obj_name} "
-        super().__init__(self.message)
-
-
 class CustomFieldBulkDeleteView(generic.BulkDeleteView):
     queryset = CustomField.objects.all()
     table = tables.CustomFieldTable
 
+    def construct_custom_field_delete_tasks(self, queryset):
+        """
+        Helper method to construct a list of celery tasks to execute when bulk deleting custom fields.
+        """
+        tasks = [
+            delete_custom_field_data.si(obj.name, set(obj.content_types.values_list("pk", flat=True)))
+            for obj in queryset
+        ]
+        return tasks
+
     def perform_pre_delete(self, request, queryset):
         if not get_worker_count(request):
             messages.error(request, "Unable to run job: Celery worker process not running.")
             return redirect(self.get_return_url(request))
-        for obj in queryset:
-            try:
-                task = delete_custom_field_data.delay(obj.name, set(obj.content_types.values_list("pk", flat=True)))
-                # Wait until the custom_field_data is completely deleted
-                task.wait(timeout=None, interval=0.1)
-                if task.state == FAILURE:
-                    # raise an exception if tasks failed.
-                    raise CeleryTaskFailure(obj_name=obj.name)
-            except CeleryTaskFailure:
-                messages.error(f"Celery task failed when deleting _custom_field_data for Custom Field {obj.name}")
-                return redirect(self.get_return_url(request))
+        tasks = self.construct_custom_field_delete_tasks(queryset)
+        chain(*tasks).apply_async()
```

Comment:
Please add a brief comment here to indicate we're firing them off in the background sequentially without concern for the results!